scripts/tools/index.js

```diff
@@ -16,7 +16,7 @@ exports.formatFilename = formatFilename;
  * `createFontFace` is used to generate the actual `@font-face` declarations
  * that get written to the appropriate files.
  */
-const createFontFace = (filename, family, weight, unicode) => {
+const createFontFace = (filename, family, weight, unicode = {type: ''}) => {
```

Comment:
Do we need the default `type: ''` here, or just the object? e.g. `unicode = {}`. Just curious!